Views/InputsView.swift

```diff
@@ -355,7 +361,40 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     return assets
   }
 
+  func saveSeeds(registeredSeeds: [RegisteredSeed]) {
```

Comment:
In order to follow phyto and fertilizer treatments, prototype should be:
func saveSeeds(_ registeredSeeds: [RegisteredSeed])